### ISSUE TITLE
Enable Footer for Afaan Oromoo Page

### ISSUE THREAD

**Issue by aeroplanejane** [bbc/simorgh#3034]
**Is your feature request related to a problem? Please describe.**
We require the footer to be enabled for Afaan Oromoo with the correct translations and correct links

Please refer to the full spec in WSMEDIA-29 (linked to confluence translations is included) 

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**

**Comment by ryanmccombe** [comment]
Added footer:

![Screen Shot 2019-08-12 at 14 58 33](https://user-images.githubusercontent.com/8601460/62870543-c04ee380-bd11-11e9-8790-4f77e740a1b5.png)

Some notes:
- Fonts still wrong - worked on by another pod
- We only have 4 translated links for afaan oromoo?  Standard simorgh config has 6 - I removed the "About Us" and "Accessibility Help" links from the standard config.  The current afaan oromoo page seems to have 8 links
- I added full stops to the end of copyright text and and external link text.  ACs didn't have them, but they do seem to be standard in the current pages and the designs
- There is a full stop missing between the copyright text and external link text in the footer component.  I didn't fix this - it's a cross cutting component and this bug is currently in production like this (bbc.com/igbo)

Let me know if any of this needs changed @aeroplanejane
